src/main/java/thePackmaster/cards/dimensiongatepack3/Corset.java

```diff
@@ -0,0 +1,30 @@
+package thePackmaster.cards.dimensiongatepack3;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.dimensiongatepack.SelfDamageAction;
+import thePackmaster.cards.dimensiongateabstracts.AbstractDimensionalCardEden;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Corset extends AbstractDimensionalCardEden {
```

Comment:
Something was bugging about this card but I didn't put my finger on it until just after finishing the first pass. Corset is in a weird design space.

If you have another block card to play before Corset (which isn't too hard, most hands will have another block card), it's a common card that gives 13 block -- way above the norm. Even if you don't have another block card, it boils down to, "will playing Corset prevent more than 3 damage?" A self-damage drawback on a block card is just too direct to calculate, which makes me question whether this design is one that we want.

If we do, I'd suggest having the net block be reduced to 10 -- still higher than Leap and quite strong for a common block card.

Worth noting that the concerns for this card feel quite distinct from my feelings about a card like Mantis Strike. Self damage on attacks has entirely different implications since the connection to (preventing) HP loss is less direct. You need to kill enemies, and sometimes you have to decide if you have enough other damage that you can afford to not play the card or not. This is also shown by Hemokinesis from the base game, though that being HP loss makes it an imperfect comparison.

Reply:
I actually had it at net Leap+1 at one point during initial development of the pack and found it too weak in practice.  I had similar thoughts of "it should be net leap, then a little better", but the restriction of "must play block beforehand to not take damage" is more relevant than I expected it to be.  But I probably went a little too high on the resulting buff.  Perhaps net Leap+2 is sufficient.

Reply:
Trying it out at Leap+2 is at least within the realm of reason 😄. I don't mind giving that a shot.